### PR TITLE
Adds the ICC LVRT 'Fallow' vehicle.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
image
image
Adds the Fallow. Its an ICC vehicle (With probably very bad stats since I didn't really balance them so far) and is 2x2 in size.
It is a light recon vehicle, so its very fast and nimble. It can be armed with either a 30mm SARDEN Autocannon with awful ROF, good damage and 7 round clips, or a 76mm low velocity cannon (with HEAT and HE) as well as a coaxial MG in place of the standard pintle secondary.
You can try it out in Valhalla.
It comes in funny camo only.
## Why It's Good For The Game
Adds more variety of non-marine vehicles to the game. And more vehicle variety in general.
## Changelog
:cl:
add: Added new ICC Recon Vehicle 'Fallow'. You can try it out in Valhalla. It has no other spawn areas.
/:cl:

### CODE REVIEW

**Comment by despong** [comment]
we're so fucking back

**Comment by Looking4bros** [comment]
add a new ERT shuttle that can be crashed into the ship like the Alamo does on Hijack and you got a fun little surprise. :)

**Comment by Aporhtonoma** [comment]
coaxial MG? like actual mg used by an actual normal marine from outside? THATS AWESOME

**Comment by lbnesquik** [comment]
> coaxial MG? like actual mg used by an actual normal marine from outside? THATS AWESOME

Coaxial means it shares the axis of the main gun. 
Something like this.

**Review by Kuro020 — APPROVED** [review]

**Comment by despong** [comment]
i hope you find out how to offset the turret because its going to look really silly when the turret faces n/s while the hull is facing w/e

File: code/modules/vehicles/armored/icc_recontank.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		/obj/structure/largecrate,
+		/obj/structure/closet/crate,
+	)
+
+/obj/vehicle/sealed/armored/multitile/icc_lvrt/enter_locations(atom/movable/entering_thing)
+	return list(get_step(src, REVERSE_DIR(dir)))
+
+/obj/vehicle/sealed/armored/multitile/icc_lvrt/Initialize(mapload)
+	. = ..()
+	var/obj/item/armored_weapon/gun = new /obj/item/armored_weapon/icc_coaxial()
```
Comment (ivanmixo) [inline code/modules/vehicles/armored/icc_recontank.dm:35]:
```suggestion
	var/obj/item/armored_weapon/icc_coaxial/gun = new(src)
	gun.attach(src)
```

**Review by ivanmixo — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**